src/desktop/tftp-server.desktop

```diff
@@ -13,7 +13,7 @@ X-SuSE-YaST-AutoInst=all
 X-SuSE-YaST-Geometry=
 X-SuSE-YaST-SortKey=
 X-SuSE-YaST-AutoInstResource=tftp-server
-X-SuSE-YaST-Keywords=TFTP,server,boot,trivial,file,transfer,protocol
+X-SuSE-YaST-Keywords=TFTP,server,boot,trivial,file,transfer,protocol,network,ftp
```

Comment:
Um, I'd rather avoid `ftp` here, it's a completely different protocol and we also have the `yast2-ftp-server` module, this could be confusing I think...

BTW there is `TFTP` at the beginning, is the keyword search case sensitive or not?